js/data.js

```diff
@@ -20,13 +20,28 @@ var exercises = {
   core: {
     abs: [],
     obliques: []
-  }
+  },
+
+  delts: [],
+  chest: [],
+  triceps: [],
+  biceps: [],
+  traps: [],
+  lats: [],
+  quads: [],
+  hams: [],
+  glutes: [],
+  abs: [],
+  obliques: [],
 };
 
 window.addEventListener('load', fetchExercises);
```

Comment:
You should move this line through line105 to `main.js`. They aren't related to your data model or storing your data model in localStorage, which is the chief use of the `data.js` file